crates/worker/src/partition/state_machine/command_interpreter/tests.rs

```diff
@@ -1,953 +0,0 @@
-            Bytes::default(),
-        ))],
-    );
-
-    state_machine
-        .on_apply(
-            Command::InvokerEffect(InvokerEffect {
-                invocation_id: caller_invocation_id,
-                kind: EffectKind::JournalEntry {
-                    entry_index: 1,
-                    entry,
-                },
-            }),
-            &mut effects,
-            &mut state_reader,
-        )
-        .await
-        .unwrap();
-    let_assert!(Effect::EnqueueIntoOutbox { message, .. } = effects.drain().next().unwrap());
-    let_assert!(
-        OutboxMessage::ServiceResponse(InvocationResponse {
-            id,
-            entry_index,
-            result: ResponseResult::Success(_),
-        }) = message
-    );
-    assert_eq!(id, callee_invocation_id);
-    assert_eq!(entry_index, 1);
-}
-
-#[test(tokio::test)]
-async fn awakeable_with_failure() {
-    let partition_key_range = PartitionKey::MIN..=PartitionKey::MAX;
-    let mut state_machine: CommandInterpreter<ProtobufRawEntryCodec> =
-        CommandInterpreter::<ProtobufRawEntryCodec>::new(
-            0,
-            0,
-            None,
-            partition_key_range,
-            SourceTable::New,
-        );
-    let mut effects = Effects::default();
-    let mut state_reader = StateReaderMock::default();
-
-    let callee_invocation_id = InvocationId::mock_random();
-    let entry = ProtobufRawEntryCodec::serialize_enriched(Entry::CompleteAwakeable(
-        CompleteAwakeableEntry {
-            id: AwakeableIdentifier::new(callee_invocation_id, 1)
-                .to_string()
-                .into(),
-            result: EntryResult::Failure(codes::BAD_REQUEST, "Some failure".into()),
-        },
-    ));
-
-    let caller_invocation_id = state_reader.register_invoked_status_and_locked(
-        InvocationTarget::mock_virtual_object(),
-        vec![JournalEntry::Entry(EnrichedRawEntry::new(
-            EnrichedEntryHeader::Awakeable {
-                is_completed: false,
-            },
-            Bytes::default(),
-        ))],
-    );
-
-    state_machine
-        .on_apply(
-            Command::InvokerEffect(InvokerEffect {
-                invocation_id: caller_invocation_id,
-                kind: EffectKind::JournalEntry {
-                    entry_index: 1,
-                    entry,
-                },
-            }),
-            &mut effects,
-            &mut state_reader,
-        )
-        .await
-        .unwrap();
-    let_assert!(Effect::EnqueueIntoOutbox { message, .. } = effects.drain().next().unwrap());
-    let_assert!(
-        OutboxMessage::ServiceResponse(InvocationResponse {
-            id,
-            entry_index,
-            result: ResponseResult::Failure(failure),
-        }) = message
-    );
-    assert_eq!(id, callee_invocation_id);
-    assert_eq!(entry_index, 1);
-    assert_eq!(failure.message(), "Some failure");
-}
-
-#[test(tokio::test)]
-async fn send_response_using_invocation_id() {
```

Comment:
Which test covers this test case?

Reply:
Basically every test where we send completions, which is a good part of the other tests...